.github/workflows/build.yml

```
@@ -81,10 +106,9 @@
           components: clippy
 
       - name: Run clippy
-        uses: actions-rs/cargo@v1
-        with:
-          command: clippy
-          args: --tests -- -D warnings --message-format=json &> clippy_report.json
+        shell: bask
```

actionlint (1.7.7)

shell name "bask" is invalid. available names are "bash", "pwsh", "python", "sh"

(shell-name)